tests/greenheart/test_hydrogen/test_PEM_costs_custom.py

```diff
@@ -0,0 +1,24 @@
+from pytest import approx
+
+from greenheart.simulation.technologies.hydrogen.electrolysis.PEM_costs_custom import (
+    calc_custom_electrolysis_capex_fom,
+)
+
+
+TOL = 1e-3
```

Comment:
This tolerance is pretty loose, I would suggest going with <=1E-6 unless there is a good reason not to